src/features/dashboard/components/Dashboard/Dashboard.jsx

```
@@ -43,7 +46,7 @@ const Dashboard = () => {
   const dashboardNames = dashboards.map((dashboard) => dashboard.name);
 
   return (
-    <ModalManager modals={modals}>
+    <useModalManager modals={modals}>
```

ESLint

[error] Unknown property 'modals' found

(react/no-unknown-property)